src/main/kotlin/study/step2/Calculator.kt

```diff
@@ -0,0 +1,41 @@
+package study.step2
+
+import java.lang.IllegalArgumentException
+import java.util.Stack
+
+class Calculator {
+    fun calculate(input: String): Int {
+        val operandStack = Stack<Int>()
+        val operatorStack = Stack<Operator>()
+
+        val inputSplit = input.split(" ")
+
+        inputSplit.forEach { s: String ->
+            when {
+                s.toIntOrNull() != null && operandStack.isEmpty() -> operandStack.add(s.toInt())
+                Operator.values().map { o -> o.operator }.contains(s) -> addOperatorStack(operatorStack, s)
+                s.toIntOrNull() != null -> operandStack.add(process(operandStack, operatorStack, s.toInt()))
+                else -> throw IllegalArgumentException()
+            }
+        }
+
+        return operandStack.pop()
+    }
+
+    private fun addOperatorStack(operatorStack: Stack<Operator>, s: String) {
+        if (!operatorStack.empty()) {
+            throw IllegalArgumentException()
+        }
+
+        operatorStack.add(Operator.values().find { o -> o.operator == s })
```

Comment:
```suggestion
        operatorStack.add(Operator.find(s))
```
위와 같이 구현할 수 있게 Operator를 찾는 로직은 Operator 내부에 두면 어떨까요?
Operator 관련 로직들은 스스로 처리해도 좋을 것 같아요.

Reply:
Operator에 find, contains 함수 추가 하였습니다.